api/credential_test.go

```diff
@@ -21,13 +21,28 @@ import (
 )
 
 func TestCredentialsHandler(t *testing.T) {
-	conf := internal.NewConfigurationWithDefaults()
-	viper.Set(configuration.ViperKeyMutatorIDTokenJWKSURL, "file://../test/stub/jwks-rsa-multiple.json")
+	conf := internal.NewConfigurationWithDefaults(configx.SkipValidation())
+	conf.SetForTest(t, configuration.MutatorIDTokenJWKSURL, "file://../test/stub/jwks-rsa-multiple.json")
+	conf.SetForTest(t, configuration.AuthenticatorAnonymousIsEnabled, true)
+	conf.SetForTest(t, configuration.AuthorizerAllowIsEnabled, true)
+	conf.SetForTest(t, configuration.MutatorIDTokenIsEnabled, true)
+
```

Comment:
I assume the defaults are disabled now in the tests?

Reply:
Removed.